### PR TITLE
Added heron_control, changed heron_description to use with uuv_simulator

### PR DESCRIPTION
**Pull request by guystoppi**

### CODE REVIEW

File: heron_description/CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 Changelog for package heron_description
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.4.0 (2018-06-26)
```
Comment (tonybaltovski) [inline heron_description/CHANGELOG.rst]: Remove this, it get automatically generated when you use `catkin_generate_changelog`

File: heron_description/package.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>heron_description</name>
-  <version>0.3.0</version>
+  <version>0.4.0</version>
```
Comment (tonybaltovski) [inline heron_description/package.xml]: Revert, this is done using `catkin_prepare_release`

File: heron_description/urdf/sensors.xacro [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+     <frameId>$(arg namespace)/imu_link</frameId>
+     <topicName>/$(arg namespace)/imu/data_raw</topicName>
+     <accelDrift>0.005 0.005 0.005</accelDrift>
+     <accelGaussianNoise>0.005 0.005 0.005</accelGaussianNoise>
+     <rateDrift>0.005 0.005 0.005 </rateDrift>
+     <rateGaussianNoise>0.005 0.005 0.005 </rateGaussianNoise>
+     <headingDrift>0.005</headingDrift>
+     <headingGaussianNoise>0.005</headingGaussianNoise>
+   </plugin>
+ </gazebo>
```
Comment (tonybaltovski) [inline heron_description/urdf/sensors.xacro]: Spacing is off.

File: heron_description/CMakeLists.txt [hunk 4]
```diff
@@ -1,9 +1,13 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(heron_description)
 
-find_package(catkin REQUIRED COMPONENTS roslaunch)
+find_package(catkin REQUIRED COMPONENTS
+    heron_msgs
```
Comment (tonybaltovski) [inline heron_description/CMakeLists.txt]: Are `heron_msgs` needed?  I don't see them used any where.

**Review by tonybaltovski — CHANGES_REQUESTED** [review]

File: heron_description/urdf/base.xacro [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    mesh with the origin of the mesh on the same position of the center of mass, otherwise
+    the mesh pose will have to be corrected below in the <visual> block.
+    Open the meshes for the RexROV vehicle in Blender to see an example on the mesh placement.
+  -->
+  <xacro:property name="visual_mesh_file" value="file://$(find heron_description)/meshes/heron_base.stl"/>
+
+  <!-- Collision geometry mesh, usually in STL format (it is recommended to keep
+  this geometry as simple as possible to improve the performance the physics engine
+  regarding the computation of collision forces) -->
+  <xacro:property name="collision_mesh_file" value="file://$(find heron_description)/meshes/heron_base.stl"/>
```
Comment (dniewinski) [inline heron_description/urdf/base.xacro]: Does the simulator use the collision mesh to calculate buoyancy?  If so, simplifying this mesh may go a long way for efficiency

File: heron_description/CMakeLists.txt (hunk 4 above)
Comment (guystoppi) [reply heron_description/CMakeLists.txt]: Should I put "roslaunch" back in?

**Review by tonybaltovski — APPROVED** [review]

**Comment by tonybaltovski** [comment]
@dniewinski any remaining concerns?

**Comment by david0429** [comment]
None on my end

**Comment by dniewinski** [comment]
LGTM

**Comment by tonybaltovski** [comment]
@Nirzvi1 can you rebase?

**Pull request merged**